client/.env.local.example

```diff
@@ -0,0 +1,5 @@
+NEXT_PUBLIC_GOOGLE_CLIENT_ID=<client id here>
+NEXT_PUBLIC_GOOGLE_API_KEY=<api key here>
+# will need to be assigned new values on deployment
+NEXT_PUBLIC_TOKEN_AUTH_URL=http://localhost:5000/auth/google/token
+NEXT_PUBLIC_IS_AUTH_URL=http://localhost:5000/auth/is-auth/
```

Comment:
why do these need to be environment variables?

Reply:
Depending on deployment. I wasn't sure if the url for the api would change in production. If not, then I guess we could hardcode them.

Reply:
It should be relative regardless, right? Eg `/auth/google/token` should work regardless of what the host is

Reply:
Frontend is on 3000 and the server is on 5000, so I thought I needed to distinguish between the two.

Reply:
ahhhhhhhhhhhhh right.  in that case, the client should just set like `baseUrl` to be whatever its meant to be, and then in the code it should just be ``${baseUrl}/auth/google/token``

although i don't know why they are separate (and don't agree with it), so i'll look to change that soon